wiremock/mappings/HmppsAuthAndDelius_GetUser_TEMPORARY-ACCOMMODATION-E2E-REFERRER.json

```diff
@@ -0,0 +1,23 @@
+{
+  "priority": 2,
+  "request": {
+    "method": "GET",
+    "urlPattern": "(?i)\/user\/TEMPORARY-ACCOMMODATION-E2E-REFERRER"
+  },
+  "response": {
+    "transformers": ["response-template"],
+    "status": 200,
+    "jsonBody": {
+      "userId": 2500077027,
+      "username": "TEMPORARY-ACCOMMODATION-E2E-REFERRER",
+      "firstName": "temporary-accommodation-e2e-referrer",
+      "surname": "accommodation-e2e-tester",
+      "email": "accomm-e2e-referrer@example.digital.justice.gov.uk",
+      "enabled": true,
+      "roles": ["ROLE_PROBATION"]
```

Comment:
I dont think we use "ROLE_PROBATION", I think we only use `CAS3_REFERRER` ,`CAS3_ASSESSOR` and `CAS3_REPORTER`

Reply:
good spot - thanks!
I'll use `CAS3_REFERRER` then as this is the referrer user

Reply:
@aliuk2012 I ended up adding all three roles to both users. I did this because when I changed this user to have `CAS3_REFERRER` only the e2e tests broke. 
If we are losing coverage here (user perms coverage I mean) we can do a deeper dive but I was not sure if it was worth it undeer the circumstances as this is only affecting this local way of running e2e against upstream services?
The alternative is, I suppose, that we look up these users' roles on the dev env (to see what their perms are for real). I might need a hand with this if you think we need to do this :)

Reply:
ah okay, will stick with `ROLE_PROBATION` then. Thanks @daveawc

Reply:
🤦  far to many different "roles"